acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/lotus/FilecoinKeyType.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.eth2signer.dsl.lotus;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public enum FilecoinKeyType {
```

Comment:
can we not reuse the KeyType class? (not actually sure if we can, just want to make sure)

Reply:
Probably not, this is feed into FC walletImport ... which uses lowercase bls and secp256k1